game-core/src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -398,6 +402,34 @@ public BattleRecordsList getBattleRecordsList() {
     return battleRecordsList;
   }
 
+  @Override
+  public PoliticsDelegate getPoliticsDelegate() {
```

Comment:
These 5 methods were moved from `DelegateFinder` and modified slightly to not be static methods.  All of the other changes in this PR are edits to use these getters instead of the ones in `DelegateFinder`.